collect_app/src/main/java/org/odk/collect/android/spatial/TileSourceFactory.java

```diff
@@ -0,0 +1,38 @@
+package org.odk.collect.android.spatial;
+
+import org.osmdroid.ResourceProxy;
+import org.osmdroid.tileprovider.MapTile;
+import org.osmdroid.tileprovider.tilesource.OnlineTileSourceBase;
+import org.osmdroid.tileprovider.tilesource.XYTileSource;
+
+/**
+  * This factory augments Tile sources provides additional tile sources, some of which
+  * are available in new version of osmdroid but not in currently used 4.2
+ */
+public class TileSourceFactory {
+    public static final OnlineTileSourceBase USGS_TOPO = new OnlineTileSourceBase(
+            "USGS National Map Topo",
```

Comment:
I've refactored TileSourceFactory to now pull up those strings from resources (couldnt find how they are used either).